.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+php:
+  - 5.6
+  - hhvm
```

Comment:
Can you remove the `php` section and add those version directly into the matrix?